esmvaltool/recipes/recipe_ocean_multimap.yml

```diff
@@ -0,0 +1,82 @@
+# ESMValTool
+# recipe_ocean_multimap.yml
+---
+documentation:
+  description: |
+    Recipe to evaluate multiple model bias against a reference dataset
+    Written by Tomas Lovato, CMCC, tomas.lovato@cmcc.it
+
+  authors:
+    - lova_to
+  
+  maintainer:
+    - lova_to
+
+  references:
+    - acknow_project
+
+  projects:
+    - crescendo
+
+
+datasets:
+# working datasets
+# CMIP5
+  - {dataset: CanESM2,       project: CMIP5,  exp: historical,  ensemble: r1i1p1,  start_year: 1995,  end_year: 2004}
+  - {dataset: CESM1-BGC,     project: CMIP5,  exp: historical,  ensemble: r1i1p1,  start_year: 1995,  end_year: 2004}
+  - {dataset: IPSL-CM5A-LR,  project: CMIP5,  exp: historical,  ensemble: r1i1p1,  start_year: 1995,  end_year: 2004}
+  #- {dataset: IPSL-CM5A-MR,  project: CMIP5,  exp: historical,  ensemble: r3i1p1,  start_year: 1995,  end_year: 2004}
```

Comment:
Please uncomment or remove, but do not leave commented out stuff lying around

Reply:
ok

Reply:
I commented the other to have the script working more quickly ... I'll remove commits at the end of this PR revision process !

Reply:
I put back the dataset into the recipe!